### PR TITLE
docs/op-guide: add TiKV performance tuning

### PR DESCRIPTION
**Pull request by lilin90**
Signed-off-by: lilin90 <lilin@pingcap.com>

## What have you changed? (mandatory)

Add a file about TiKV performance tuning.

## Refer to a related PR or issue link (optional)

Move the following document from pingcap/docs: https://github.com/pingcap/docs/blob/master/op-guide/tune-tikv.md

### CODE REVIEW

File: docs/op-guide/tune-tikv.md [hunk 1]
```diff
@@ -0,0 +1,272 @@
...
+# The number of gRPC connections between each TiKV instance
+grpc-raft-conn-num = 10
+
+# Most read requests from TiDB are sent to the coprocessor of TiKV. This parameter is used
+# to set the number of threads of the coprocessor. If many read requests exist, add the
+# number of threads and keep the number within that of the system CPU cores. For example,
+# for a 32-core machine deployed with TiKV, you can even set this parameter to 30 in
+# repeatable read scenarios. If this parameter is not set, TiKV automatically sets it to
+# CPU cores * 0.8.
+end-point-concurrency = 8
```
Comment (siddontang) [inline docs/op-guide/tune-tikv.md]: seem this is changed now. @breeswish

Comment (lilin90) [reply docs/op-guide/tune-tikv.md]: OK. Then close this PR? @breeswish Could you help rewrite this doc?

Comment (breeswish) [reply docs/op-guide/tune-tikv.md]: Yes, I will share it to you once complete.

Comment (lilin90) [reply docs/op-guide/tune-tikv.md]: @breeswish Thanks!

**Pull request closed without merging**